src/menus/icon-menu.jsx

```diff
@@ -114,12 +108,6 @@ const IconMenu = React.createClass({
      * horizontal: [left, center, right].
      */
     targetOrigin: PropTypes.origin,
-
-    /**
-     * Sets the delay in milliseconds before closing the
-     * menu when an item is clicked.
-     */
-    touchTapCloseDelay: React.PropTypes.number,
```

Comment:
This was used by the old `menu` components. The right way to implement it now is to use the `IconMenu` as a controlled component and to delay the effect of `onRequestChange`.